test/unit/controllers/Irc_EntryController.test.js

```diff
@@ -177,23 +195,25 @@ describe('UNIT Irc_EntryController', () => {
     });
 
     after(() => {
+      clock.restore();
       global.Centres = original_centre;
     });
 
-    it('should update the centre', () =>
-	expect(global.Centres.update).to.be.calledWith(
+    it('should update the centre', () => {
```

Comment:
You're quite right, that was a leftover. I began by `var`'ing up the `clock` in the `it` function definition and without the curlies, I was getting errors